code/script.js

```diff
@@ -1,10 +1,79 @@
 // Variables that point to selected DOM elements
 const chat = document.getElementById('chat');
+const form = document.getElementById('name-form');
+const nameInput = document.getElementById('name-input');
+const inputWrapper = document.getElementById('input-wrapper');
 
-// If you need any global variables that you can use across different functions, declare them here:
 
+// Function handling name input
+
+const handleNameInput = (event) => {
+  event.preventDefault()
+  const name = nameInput.value
+  showMessage(name, 'user')
+  nameInput.value = ''
+
+  setTimeout(() => likingMusic(name), 1000 )
+}
+
+//event listner method is starting function handleNameInput
+form.addEventListener("submit", handleNameInput);
+
+//starting question
+const likingMusic = (name) => {
+  showMessage(`Hi ${name}. Do you like listening to music?`, 'bot')
+  setTimeout(() => yesOrNo(), 100)
+}
+// two options to choose from
+
+const yesOrNo = () => {
+  inputWrapper.innerHTML = `
+  <button id="optionOne">YES</button>
+  <button id="optionTwo">NO</button>
+  `
+  document.getElementById('optionOne').addEventListener("click", () => {
```

Comment:
//you could add the user answer on screen in the conversation//
showMessage('Yes', 'user')